docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source.md

```diff
@@ -5,10 +5,16 @@ sidebar_label: Gmail Trace Logs
 description: The Sumo Logic Gmail Trace Logs source pulls the Gmail log from the BigQuery using BigQuery Library APIs and ingests them into the Sumo Logic to store, analyze, and alert.
 ---
 
+
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('img/send-data/gmail-trace-logs-icon.svg')} alt="Gmail" width="50"/>
 
+:::note
+This source was originally developed for [Gmail logs in BigQuery](https://support.google.com/a/topic/7233311?fl=1&sjid=293657922002214011-NA), which is now replaced by [Google Workspace logs and reports in BigQuery](https://support.google.com/a/topic/9079469?fl=1&sjid=293657922002214011-NA).
+Please use our [Google BigQuery source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source), see Example 3, instead
```

Comment:
```suggestion
This source was originally developed for [Gmail logs in BigQuery](https://support.google.com/a/topic/7233311?fl=1&sjid=293657922002214011-NA), which has been replaced by [Google Workspace logs and reports in BigQuery](https://support.google.com/a/topic/9079469?fl=1&sjid=293657922002214011-NA).

Please use our [Google BigQuery source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source) instead.
```

Reply:
@duchatran you mentioned `Example 3` - what are you referring to?

With this note, we are redirecting users to BigQuery source. In what scenario(s) should customers continue to use  Gmail Trace Logs source? Or is it now obsolete?

Reply:
Gmail Trace Logs no longer works for customers who setup Gmail logs anew. I am just not sure if it still works for existing customers so decided to leave this page out with this note

Reply:
Oh I overlooked at didn't recognize you removed "Example 3". That source is a more generic BigQuery source, and Example 3 is for Gmail specifically.

Reply:
(Added it back in the latest commit)